### DISCUSSION TITLE
Wrapping additional Spotify Web APIs

### DISCUSSION THREAD

**Discussion by e13h** [CyanBook/spotipy2#2] [Ideas]
I like what I've seen so far in `spotipy2`, and I'm curious to know if there are plans to wrap other Spotify Web APIs. Some features I am specifically interested in are:
* [Player API](https://developer.spotify.com/documentation/web-api/reference/#category-player)
  * [Start/resume playback](https://developer.spotify.com/documentation/web-api/reference/#endpoint-start-a-users-playback), [pause](https://developer.spotify.com/documentation/web-api/reference/#endpoint-pause-a-users-playback), [seek](https://developer.spotify.com/documentation/web-api/reference/#endpoint-seek-to-position-in-currently-playing-track), [get information about current playback](https://developer.spotify.com/documentation/web-api/reference/#endpoint-get-information-about-the-users-current-playback), etc.
* Playlist modification
  * [Add items to playlist](https://developer.spotify.com/documentation/web-api/reference/#endpoint-add-tracks-to-playlist), [create a playlist](https://developer.spotify.com/documentation/web-api/reference/#endpoint-create-playlist), etc.

I look forward to finding ways to contribute, so I'm happy to open a PR for these features, but I don't want to work on something that is already in progress! :laughing:

**Comment by CyanBook** [comment]
Hi!
No, i'm not currently working to these features! Feel free to help, it's much appreciated.

Actually i've never have someone open a PR, so here's just a couple of guidelines.
- Use type hinting when possible
- Try to follow the current design, shouldn't be a big problem
- Use async where necessary

For anything, you can comment here or contact me on [Telegram](t.me/cyanbook)!
